docker/fusionauth/docker-compose.yml

```diff
@@ -65,6 +65,11 @@ services:
       FUSIONAUTH_APP_URL: http://fusionauth:9011
       SEARCH_SERVERS: http://search:9200
       SEARCH_TYPE: elasticsearch
+    healthcheck:
```

Comment:
Should we use `/api/status` to check the various DB connection status codes as well? 
https://fusionauth.io/docs/apis/system#retrieve-system-status

Reply:
Done, thanks